[2025-04-18T06:47:15.684+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-04-18T06:47:15.698+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: image_processing_pipeline.list_new_images scheduled__2025-04-18T06:35:00+00:00 [queued]>
[2025-04-18T06:47:15.705+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: image_processing_pipeline.list_new_images scheduled__2025-04-18T06:35:00+00:00 [queued]>
[2025-04-18T06:47:15.705+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 2
[2025-04-18T06:47:15.715+0000] {taskinstance.py:2890} INFO - Executing <Task(S3ListOperator): list_new_images> on 2025-04-18 06:35:00+00:00
[2025-04-18T06:47:15.721+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'image_processing_pipeline', 'list_new_images', 'scheduled__2025-04-18T06:35:00+00:00', '--job-id', '48', '--raw', '--subdir', 'DAGS_FOLDER/image_processing_dag.py', '--cfg-path', '/tmp/tmpl1ec0hz4']
[2025-04-18T06:47:15.723+0000] {standard_task_runner.py:105} INFO - Job 48: Subtask list_new_images
[2025-04-18T06:47:15.723+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=310) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-04-18T06:47:15.724+0000] {standard_task_runner.py:72} INFO - Started process 311 to run task
[2025-04-18T06:47:15.749+0000] {task_command.py:467} INFO - Running <TaskInstance: image_processing_pipeline.list_new_images scheduled__2025-04-18T06:35:00+00:00 [running]> on host 96ec7c07bc54
[2025-04-18T06:47:15.801+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='image_processing_pipeline' AIRFLOW_CTX_TASK_ID='list_new_images' AIRFLOW_CTX_EXECUTION_DATE='2025-04-18T06:35:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-04-18T06:35:00+00:00'
[2025-04-18T06:47:15.801+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-04-18T06:47:15.810+0000] {s3.py:837} INFO - Getting the list of files from bucket: original-images-*** in prefix:  (Delimiter /)
[2025-04-18T06:47:15.819+0000] {base.py:84} INFO - Retrieving connection 'aws_default'
[2025-04-18T06:47:15.819+0000] {connection_wrapper.py:325} INFO - AWS Connection (conn_id='aws_default', conn_type='aws') credentials retrieved from login and password.
[2025-04-18T06:47:16.361+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-04-18T06:47:16.362+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=image_processing_pipeline, task_id=list_new_images, run_id=scheduled__2025-04-18T06:35:00+00:00, execution_date=20250418T063500, start_date=20250418T064715, end_date=20250418T064716
[2025-04-18T06:47:16.418+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-04-18T06:47:16.452+0000] {taskinstance.py:3901} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-04-18T06:47:16.453+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
